src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -18,8 +19,43 @@ using namespace winrt::Windows::Storage::AccessCache;
 using namespace winrt::Windows::Storage::Pickers;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
 
+static const std::set<winrt::guid> InBoxProfileGuids{
```

Comment:
please try to avoid statically initialized stl types. just like you don't have a `static hstring`, don't have a `static set`.

Use a constexpr std::array or something.